### ISSUE TITLE
 Is there any plan to update Atlas to work on ios16 serotonin?

### ISSUE THREAD

**Issue by L-skyz** [Chr1sDev/Atlas#6]
hi
Is there any plan to update Atlas to work on ios16 serotonin?
I converted it to rootless and installed it but it doesn't work.

**Comment by ChristopherA8** [comment]
yeah, I was in the process of doing that a couple of weeks ago. Got a lot of schoolwork though so we'll see if I finish that soon

**Comment by L-skyz** [comment]
Atlas Lite was released. 
Will you not update the original features of Atlas anymore?
Or was Atlas Lite released as a standalone tweak specialized for double tap skipping?

**Comment by ChristopherA8** [comment]
Yes and no, I’m still waiting for an update to cephei so I can do proper tweak preferences which limits a lot of features I can add without having settings to turn them off or on, and also the redesign aspect seems useless to me at this point just cause apple already redesigned their video player in 16 and it looks pretty good all things considered. If cephei comes out with an update I’ll definitely consider updating a lot of tweaks for rootless but otherwise it’s just this for now :/ unless someone wants to do it for me 👀lol
